data/css/_colors.scss

```diff
@@ -7,10 +7,3 @@ $accent-dark-color: #cc532b !default;
 
 $background-light-color: #e9e9e3 !default;
 $background-dark-color: #ddddd5 !default;
-
-$white-60-color: transparentize(white, 0.4) !default;
```

Comment:
OK. like I said, I made the same case to @tchx84 as you are making now, so check with him to see if there's nothing that I'm missing.